metaci/api/views/robot.py

```diff
@@ -0,0 +1,33 @@
+from rest_framework.renderers import JSONRenderer, BrowsableAPIRenderer
+from rest_framework import viewsets
+from django_filters.rest_framework import DjangoFilterBackend
+from metaci.api.renderers import SimpleCSVRenderer
+from metaci.api.serializers.robot import RobotTestResultSerializer
+from metaci.testresults.models import TestResult
+from metaci.testresults.filters import RobotResultFilter
+from rest_framework.filters import OrderingFilter
+
+
+class RobotTestResultViewSet(viewsets.ReadOnlyModelViewSet):
+    """
+    A viewset for viewing robot framework test results
+    """
+
+    serializer_class = RobotTestResultSerializer
+    renderer_classes = [BrowsableAPIRenderer, JSONRenderer, SimpleCSVRenderer]
+    filterset_class = RobotResultFilter
+
+    def get_queryset(self):
+        queryset = TestResult.objects.filter(method__testclass__test_type="Robot")
```

Comment:
We probably need some sort of permission filter so that a user can only see test results for planrepos that they have the "view builds" permission on.

Reply:
@boakley I still don't see anything in here to make sure that the queryset only includes test results for builds that the user has permission to view. Probably something like adding a filter for `build_flow__build__planrepo__in=PlanRepository.objects.for_user(self.request.user)`